scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -333,12 +355,43 @@
         {
             "condition": "[parameters('useAzureMonitorPrivateLinkScope')]",
             "type": "Microsoft.Resources/deployments",
-            "name": "[Concat('aks-monitoring-msi-ampls-scope', '-',  uniqueString(parameters('aksResourceId')))]",
+            "name": "[Concat('aks-monitoring-msi-ampls-scope-config', '-',  uniqueString(parameters('aksResourceId')))]",
+            "apiVersion": "2017-05-10",
```

Comment:
This api version is old, should we consider using the new ones?

Reply:
> This api version is old, should we consider using the new ones?

We dont have dependency on the API version and I think, we should be good using GA API version.